RC SAML SSO updates (#1994)
RedisLabs/redislabs-docs: content/rc/security/single-sign-on/saml-sso.md

        |---------------|-------------|
        | EntityDescriptor's **entityID** | Audience URI |
        | AssertionConsumerService's **Location** | Hub ACS URL |

    To learn more about how to configure service provider apps, see your identity provider's documentation.

### Activate SAML SSO {#activate-saml-sso}

After you finish the required SAML SSO configuration between your identity provider and Redis Cloud account, you can test and activate SAML SSO:

1. For **Activate SAML integration**, select the **Activate** button.

1. From the **Logout notification** dialog, select **Continue**. This redirects you to your configured identity provider's sign-in screen.

1. Sign in with your identity provider.

1. When redirected to the Redis Cloud sign-in screen, select **Sign in with SSO** and enter your credentials.

    {{<image filename="images/rc/button-sign-in-sso.png" alt="Sign in with SSO button">}}{{</image>}}

    {{<note>}}
If you see a **SAML activation failed** notification when redirected to the Redis Cloud sign-in screen, sign in with your previous credentials and review your SAML configuration for issues.
    {{</note>}}

## Add SAML users {#add-saml-users}

> To add a user to an account and enable SAML SSO for them:

Reviewer: ```suggestion
To add a user to an account with SAML SSO enabled:
```

1. From your identity provider's admin console, [add a new user](https://help.okta.com/en/prod/Content/Topics/users-groups-profiles/usgp-add-users.htm) or edit an existing user's profile.

1. Enter the Redis Cloud account ID and a [user role]({{<relref "/rc/administration/access-management#team-management-roles">}}) in the **redisAccountMapping** field.

    You can add the same user to multiple SAML-enabled accounts with a comma-separated list: 

    12345=owner,54321=manager

1. [Assign the Redis Cloud SAML integration app](https://help.okta.com/en/prod/Content/Topics/users-groups-profiles/usgp-assign-apps.htm) to the user.

1. The user can now sign in to Redis Cloud with SAML SSO.

To learn how to manage SAML user profiles, see your identity provider's documentation.

## Bind other accounts

> After you set up SAML SSO for one account, you can edit the SAML configuration to bind other accounts where you are an account owner. This lets you use the same domain for SSO across multiple accounts.

Reviewer: ```suggestion
After you set up SAML SSO for one account, you can edit the SAML configuration to bind other accounts you own to the existing SAML config. This lets you use the same domain for SSO across multiple accounts.
```

To bind other accounts to an existing SAML SSO configuration:

1. Go to **Access Management > Single Sign-on** in the Redis Cloud [admin console](https://app.redislabs.com).

1. Select the **Edit** button.

1. For **Bind other accounts to SAML config**, select the checkboxes for the other accounts you want to bind to SAML SSO.

    {{<image filename="images/rc/access-management-saml-bind-accounts.png"  alt="Bind other accounts to SAML configuration screen.">}}{{</image>}}

1. Select **Save**.

1. From the **Bind accounts** dialog, select **Continue** to enable SAML SSO for the selected accounts.

## Deactivate SAML SSO

> A SAML-enabled account must have a local (non-SAML) user with the owner role assigned before you can deactivate SAML SSO for that account.

Reviewer: ```suggestion
Before you can deactivate SAML SSO for an account, it must have a local (non-SAML) user with the owner role assigned.
```

To deactivate SAML SSO for a specific account:

1. In the Redis Cloud [admin console](https://app.redislabs.com), select your name to display your available accounts.

> 1. If the relevant account is not already selected, you need to select it from the **Switch account** list.

Reviewer: ```suggestion
1. If the relevant account is not already selected, select it from the **Switch account** list.
```

## Deactivate SAML SSO

> Before you can deactivate SAML SSO for an account, it must have a local (non-SAML) user with the owner role assigned. You can only deactivate SAML SSO for the account you are currently signed in to.

Reviewer 2: @rrelledge to deactivate SAML you must be logged in to that specific account as a local (non-SAML) user with the owner role, i.e. even if the account has another local owner user but you are signed in as a SAML user you can still not deactivate it.

Author: Fixed.
